frontend/catalyst/compilation_pipelines.py

```diff
@@ -60,7 +60,7 @@
 
 
 # pylint: disable=too-many-return-statements
-def mlir_type_to_numpy_type(t):
+def mlir_type_to_numpy_type(t):  # pragma: no cover
```

Comment:
In fact looking at the rest of the changes most of them should be removed I think. There is no point in hiding the uncovered code unless there is a good reason for it. For example the concrete impls for the jax primitives which just raise `NotImplementedError` would be ok to mark no cover, since having tests for those wouldn't really add any value.

Reply:
:100: Fixed!